### DISCUSSION TITLE
What's new (or will be new) in the upcoming v2 release?

### DISCUSSION THREAD

**Discussion by 0x464e** [Azure/azure-functions-dotnet-worker#2862] [Feedback]
I see that `Microsoft.Azure.Functions.Worker.Sdk` and `Microsoft.Azure.Functions.Worker` will get v2 releases quite soon (preview releases are somewhat far along already).

Is there any place from where I can find info on what new features and/or changes will be in the upcoming version?

**Comment by kshyju** [comment]
Please check the release notes page - https://github.com/Azure/azure-functions-dotnet-worker/releases
